README.md

```diff
@@ -56,73 +59,59 @@ Interested in React Native project involving Bluetooth Low Energy? [We can help
 
 [Learn more about Polidea's BLE services](https://www.polidea.com/services/ble/?utm_source=Github&utm_medium=Npaid&utm_campaign=Tech_BLE&utm_term=Code&utm_content=GH_NOP_BLE_COD_RNB001).
 
-Documentation can be found [here](https://polidea.github.io/react-native-ble-plx/).
+[Documentation can be found here](https://polidea.github.io/react-native-ble-plx/).
 
 Contact us at [Polidea](https://www.polidea.com/project/?utm_source=Github&utm_medium=Npaid&utm_campaign=Kontakt&utm_term=Code&utm_content=GH_NOP_KKT_COD_RNB001).
 
 Contact us at [Gitter](https://gitter.im/RxBLELibraries/react-native-ble) if you have any questions, feedback or want to help!
 
 ## Configuration & Installation
 
-### iOS (expo/Podfile and RN 0.60+, [example setup](https://github.com/Cierpliwy/SensorTagExpo))
+### Expo
+
+1. Make sure your Expo project is ejected (formerly: detached). You can read how to do it [here](https://docs.expo.io/versions/latest/expokit/eject/) and [here](https://docs.expo.io/versions/latest/expokit/expokit). (only for expo)
+2. Follow steps for iOS/Android.
+
+### iOS ([example setup](https://github.com/Cierpliwy/SensorTag))
 
-1. Make sure your Expo project is ejected (formerly: detached). You can read how to do it [here](https://docs.expo.io/versions/v32.0.0/expokit/eject/) and [here](https://docs.expo.io/versions/latest/expokit/expokit). (only for expo)
-2. `npm install --save react-native-ble-plx`
-3. `react-native link react-native-ble-plx`
-4. Open Xcode workspace located inside `ios` folder and add empty Swift file if you don't have at least one:
+1. `npm install --save react-native-ble-plx`
+2. `npx react-native link react-native-ble-plx`
+3. Open Xcode workspace located inside `ios` folder and add empty Swift file if you don't have at least one:
    - Select File/New/File...
    - Choose Swift file and click Next.
    - Name it however you want, select your application target and create it.
    - Accept to create Objective-C bridging header.
-5. Update your `ios/Podfile` to contain:
+4. Update your `ios/Podfile` to contain (it may be already there):
    ```
    pod 'react-native-ble-plx', :path => '../node_modules/react-native-ble-plx'
    ```
-6. Enter `ios` folder and run `pod update`
-7. Minimal supported version of iOS is 8.0
-8. If you want to support background mode:
+5. Enter `ios` folder and run `pod update`
+6. Minimal supported version of iOS is 8.0
```

Comment:
I wonder whether it should be either as a prerequisite even before the first point or at the top of the readme? On the other hand Android integration needs to set `minSdkVersion = 18` which is a valid step. Here it looks like a prerequisite. Maybe make it:
```suggestion
6. Make sure minimal version of iOS is at least 8.0
```

Reply:
I'll check if it's even possible to compile RN project with iOS version less then 8.0.

Reply:
[It appears that RN is iOS > 10](https://stackoverflow.com/questions/42200831/which-version-does-react-native-support-ios-and-android)

Reply:
Let's remove it then...

Reply:
Updated @dariuszseweryn